src/iconsManifest/supportedExtensions.ts

```diff
@@ -1522,6 +1522,19 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
     },
     { icon: 'elasticbeanstalk', extensions: [], format: FileFormat.svg },
+    {
+      icon: 'eleventy',
+      extensions: ['eleventy.js'],
```

Comment:
Eleventy 2.0 also supports `eleventy.config.js`, and `eleventy.config.cjs` (in addition to `.eleventy.js`). Can those extensions be added?

11ty/eleventy#2464

Reply:
Sure! Thanks for the suggestion!

Reply:
Will tackle that in another PR as it will probably need a new icon for those files. #3222